frontend/components/CompanyBrowser.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div
+    class="flex-col w-full mx-0 flex md:flex-row-reverse h-full overflow-hidden"
+  >
+    <div class="flex bg-gray-200 w-full h-full max-h-full">
+      <MapDisplay />
+    </div>
+    <div
+      class="flex flex-col bg-gray-50 w-full md:h-full h-1/2 md:w-3/5 lg:w-3/5 xl:w-2/5 sm:h-full"
+    >
+      <CompanyListHeader />
+      <CompanyList />
+    </div>
+  </div>
+</template>
+
+<script setup>
+import MapDisplay from "./MapDisplay.vue";
```

Comment:
Also sometimes we have
`import NavigationBar from "@/components/NavigationBar.vue";`
but here we have 
`import MapDisplay from "./MapDisplay.vue";`
Please make it more consistent.

Reply:
Paths were corrected to match others